### PR TITLE
Add "commented badge" for builds where all failed jobs have at least a comment

### PR DESCRIPTION
**Pull request by Martchus**
This change is on top of https://github.com/os-autoinst/openQA/pull/3865

---

The black certificate icon was made stricter in
f4a3ee6 to require actual bugrefs or
labels instead of arbitrary comments. In the case the black certificate is
no longer shown this new "commented badge" takes its place.

See https://progress.opensuse.org/issues/91658

### CODE REVIEW

**Comment by okurz** [comment]
Please update docs accordingly as well

File: docs/UsersGuide.asciidoc [hunk 1]
```diff
@@ -298,9 +298,10 @@ image::images/review_badges.png[Review badges]
 
 ==== Meaning of the different colors ====
 
-* The green icons shows up when there is no work to be done.
 * No icon is shown if at least one failure still need to be reviewed.
-* The black icon is shown if all review work has been done.
+* The green tick shows up when there is no work to be done.
```
Comment (okurz) [inline docs/UsersGuide.asciidoc:302]: To be consistent can you say "the green tick *icon*"?

**Comment by okurz** [comment]
Please provide a screenshot as well

**Comment by Martchus** [comment]
A screenshot is contained in the last commit: https://github.com/os-autoinst/openQA/blob/53bd6559972fade3cd7a7a02d948ec8d80dc9f54/docs/images/review_badges.png

**Review by okurz — APPROVED** [review]

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**